flink-streaming-java/src/test/java/org/apache/flink/streaming/api/operators/LambdaTrigger.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.api.operators;
+
+import org.assertj.core.api.Assertions;
+
+import java.util.function.Consumer;
+
+/** Triggerable for test. */
+public class LambdaTrigger<K, N> implements Triggerable<K, N> {
```

Comment:
It might be better to improve the naming and JavaDoc of this class, since it is a shared utility class instead of a private class now.